src/main/kotlin/no/nav/familie/ef/søknad/api/DokumentasjonsbehovController.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.ef.søknad.api
+
+import no.nav.familie.ef.søknad.integration.SøknadClient
+import no.nav.familie.kontrakter.ef.søknad.dokumentasjonsbehov.DokumentasjonsbehovDto
+import no.nav.familie.sikkerhet.EksternBrukerUtils
+import no.nav.familie.sikkerhet.EksternBrukerUtils.personIdentErLikInnloggetBruker
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+@RestController
+@RequestMapping(path = ["/api/dokumentasjonsbehov"])
+@ProtectedWithClaims(issuer = EksternBrukerUtils.ISSUER, claimMap = ["acr=Level4"])
+class DokumentasjonsbehovController(private val søknadClient: SøknadClient) {
```

Comment:
Skulle vi forresten skrevet en test på denne controlleren? Mener å huske at vi begynte på noe tilsvarende i mottak, eller i hvert fall snakket om det 🤔

Reply:
8e94bf3 Jag tok ikke inn wiremock, men gjorde det slik for å få testet noe nå